paper.tex

```diff
@@ -79,6 +79,28 @@ \subsubsection{Consumer}
 \subsubsection{Broker}
 Brokers represent data providers and consumers to do computing tasks in the DLT (Distributed Ledger Technology) since brokers are expected to have high computing power. Once a data provider wants to launch a new product, the data provider requests a broker to create a new MAM channel, a new Product Contract, which specifies details about products, and upload this product to the IPFS (InterPlanetary File System)\cite{IPFS}. Also the broker certifies the session key which is used for symmetric encryption for streaming data encryption and decryption. Then the broker provides the matching service for data provider’s products and consumer’s queries. After that, the broker is requested to deal with the trading process and upload the provider’s data streams to the MAM channel.
 
+\subsection{Component}
+\subsubsection{IOTA}
+IOTA\cite{IOTAwhitepaper} is a cryptocurrency which the ledger does not consist of transactions grouped into blocks and stored in sequential chains, but a directed acyclic graph(DAG) of individual transactions, \textbf{Tangle}. To issue a transaction in the network, one needs to perform a small amount of Proof-of-Work (PoW) that verifies two previous transactions. Since every actor in Tangle has the same role, there is no need to offer transaction fees. Therefore, it is possible to make \textbf{micropayments} which the importance will increase in the rapidly developing IoT industry, and \textbf{zero-transactions} that one could store information securely within Tangle transactions. Moreover, this structure also enables high scalability of transactions that could resolve the drawbacks of blockchain and the concern of rapid growth of IoT devices.
+
+\subsubsection{smart contract}
```

Comment:
Change to Ethereum Smart Contract since Nick Szabo introduced this concept in 1994 and defined a smart contract as “a computerized transaction protocol that executes the terms of a contract.”